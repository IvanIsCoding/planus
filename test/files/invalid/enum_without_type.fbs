enum Foo { X = 0 }