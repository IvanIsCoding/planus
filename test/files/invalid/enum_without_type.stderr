[0m[1m[38;5;9merror[0m[1m: Enum declarations must have a representation-type[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/enum_without_type.fbs:1:9
  [0m[34mâ”‚[0m
[0m[34m1[0m [0m[34mâ”‚[0m enum Foo[0m[31m [0m{ X = 0 }
  [0m[34mâ”‚[0m         [0m[31m^[0m
  [0m[34mâ”‚[0m         [0m[31mâ”‚[0m
  [0m[34mâ”‚[0m         [0m[31mType should be inserted here[0m
  [0m[34mâ”‚[0m         [0m[34mHint: Try inserting `: [integer type]`[0m

Error: Bailing because of errors
